configs/llm_model_configs/azure-gpt-4o-mini.json

```diff
@@ -0,0 +1,7 @@
+{
+    "context_window": 128000,
+    "model": "gpt-4o-mini",
+    "model_endpoint_type": "azure",
+    "model_endpoint": "https://letta.openai.azure.com/openai/deployments/gpt-4o-mini/chat/completions?api-version=2023-03-15-preview",
```

Comment:
Do we want this address to be in the OSS repo? @sarahwooders

Reply:
vs being use provided (or GH actions secrets provided)

Reply:
Removed for now